operators/advanced_network/managers/dpdk/adv_network_dpdk_mgr.cpp

```diff
@@ -49,9 +50,12 @@ struct RxWorkerParams {
   int num_segs;
   uint32_t batch_size;
   struct rte_ring* ring;
+  struct rte_ring* lb_ring;          // Ring used between TX and RX when in SW loopback mode
   struct rte_mempool* flowid_pool;
-  struct rte_mempool* burst_pool;
-  struct rte_mempool* meta_pool;
+  struct rte_mempool* rx_burst_pool; // Pool used to pull out bursts from RX pool
+  struct rte_mempool* tx_burst_pool; // Pool used for loopback mode to return transmitted bursts
+  struct rte_mempool* rx_meta_pool;  // Pool used for RX metadata structures
+  struct rte_mempool* tx_meta_pool;  // Pool used in loopback for returning transmitted metadata
```

Comment:
suggestive nitpick for consideration, fine otherwise
```suggestion
  struct rte_mempool* burst_pool;        // Pool used to pull out bursts from RX pool
  struct rte_mempool* meta_pool;         // Pool used for RX metadata structures
  struct rte_mempool* sw_lb_burst_pool_; // Pool used in sw loopback mode to return transmitted bursts
  struct rte_mempool* sw_lp_meta_pool_;  // Pool used in sw loopback mode to return transmitted metadata
```